gcloud/datastore/connection.py

```diff
@@ -252,15 +256,18 @@ def run_query(self, dataset_id, query_pb, namespace=None, eventual=False):
         matching the query.
 
         You typically wouldn't use this method directly, in favor of the
-        :func:`gcloud.datastore.query.Query.fetch` method.
+        :meth:`gcloud.datastore.query.Query.fetch` method.
 
         Under the hood, the :class:`gcloud.datastore.query.Query` class
         uses this method to fetch data:
 
         >>> from gcloud import datastore
         >>> from gcloud.datastore.query import Query
-        >>> connection = datastore.get_connection()
-        >>> query = Query(dataset_id='dataset-id', 'MyKind')
+        >>>
```

Comment:
Do we interleave empty docstring example lines like this?  I think it would make more sense just to have a blank line.